README.md

```diff
@@ -15,6 +15,12 @@ development & testing:
 
     oc create -f deploy/examples/ceph-demo.yaml
 
+> If your cluster runs multiple compute nodes you need to allow TCP traffic to
```

Comment:
worker nodes

Reply:
I think compute nodes is correct, looking at "oc get nodes"?

# oc get nodes
NAME                 STATUS    ROLES          AGE       VERSION
master.example.com   Ready     infra,master   2d        v1.11.0+d4cacc0
node1.example.com    Ready     compute        2d        v1.11.0+d4cacc0
node2.example.com    Ready     compute        2d        v1.11.0+d4cacc0

Reply:
Oh, OK, so this is an OpenShift naming convention.